acceptance/bundle/templates/default-python/fail-missing-uv/output.txt

```diff
@@ -24,7 +24,7 @@ Validation OK!
 
 >>> [CLI] bundle deploy
 Building python_artifact...
-Error: build failed python_artifact, error: exit status 127, output: bash: uv: command not found
```

Comment:
This seems to be specific output to your local run and will cause tests to fail on CI, could you revert?